hooks-admin.php

```diff
@@ -2,9 +2,7 @@
 
 // Filter export formats array
 function pb_mpdf_add_to_formats( $formats ) {
-		unset( $formats['standard']['pdf'] );
-		unset( $formats['standard']['print_pdf'] );
-		$formats['standard'] = array( 'mpdf' => __( 'PDF (mPDF)', 'pressbooks' ) ) + $formats['standard'];
+		array_splice( $formats['standard'], 2, 0, array( 'mpdf' => __( 'PDF (mPDF)', 'pressbooks' ) ) );
```

Comment:
`array_splice` - keys in replacement array are not preserved. Ends up as: 

![image](https://cloud.githubusercontent.com/assets/2048170/23478980/db1e7bd6-fe77-11e6-9ab3-6645bcd59b72.png)

could just slap it on at the end `$formats['standard']['mpdf'] = __( 'PDF (mPDF)', 'pressbooks-mpdf' );`